### DISCUSSION TITLE
¿how to add or edit device type?

### DISCUSSION THREAD

**Discussion by WebTeccl** [opendcim/openDCIM#1476] [Q&A]
![image](https://github.com/opendcim/openDCIM/assets/144266280/a9af5eac-1e5e-4cb6-a3e3-087b88e7765a)

Hello, I'm just installing the opendcim software (I still need to create a "step by step" with details) but I already have it working.
I wanted to know if it is possible to edit the list of "device types" that appears.

thanks

**Comment by timlegge** [comment]
Hi

No it is not.  Those are for specific functionality more so than device
types.  You can create custom attributes in the settings.

Tim

Timothy Legge
***@***.***
***@***.***
